.babelrc

```diff
@@ -0,0 +1,18 @@
+{
+  "assumptions": {
+    "setPublicClassFields": true
+  },
+  "presets": [
+    "@babel/preset-typescript"
+  ],
+  "plugins": [
+    "@babel/plugin-proposal-export-namespace-from",
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-optional-chaining",
+    ["@babel/plugin-transform-modules-commonjs", {"noInterop": false}]
```

Comment:
Its default to `false` and deprecated. See here: https://babeljs.io/docs/en/babel-plugin-transform-modules-commonjs#nointerop